docs/linux/zbd-api.md

```diff
@@ -527,18 +524,19 @@ struct blk_zone zones[0];
 ...
 ```
 
-With this method, the main code responsible for issuing and parsing zone reports
-always has access to the `capacity` field of `struct blk_zone` regardless of
-the kernel version the code is executed on. For kernels before kernel version
-5.9, the zone capacity field will always be equal to 0, meaning that the zone
-capacity should be ignored and that the zone size should be used in its place.
-Different coding techniques can also be used to always return a zone capacity
-equal to the zone size for kernels lacking support for this field.
+If this method is used, the main code that is responsible for issuing and
+parsing zone reports always has access to the `capacity` field of `struct
+blk_zone` regardless of the kernel version the code is executed on. For kernels
+before kernel version 5.9, the zone capacity field is always equal to 0, which
```

Comment:
Looking again at that example code, it is bad... I will update this to show code that always returns zone capacity set to zone length when zone cap is not supported.

Reply:
There's no rush on this from my side, but I'll make a note of it and remind you of it in a month.